styles/desktop.css

```diff
@@ -60,6 +70,19 @@
     font-size: 48px;
     line-height: 60px;
     width: 45%;
+    animation: 3s infinite reverse translation linear;
+  }
+
+  @keyframes translation {
+    from {
+      transform: translate(0%);
+      margin-left: 0;
+    }
+
+    to {
+      transform: translate(50%);
+      margin-left: 100%;
+    }
   }
 
```

Comment:
the functionality of the hamburger menu does not work please look into this problem as shown below

[here](https://www.loom.com/share/5c80d9a612d7473da880e75ab42f42c7)